src/graphql/operations/user.ts

```diff
@@ -6,19 +6,35 @@ import { capture } from '@snapshot-labs/snapshot-sentry';
 export default async function (parent, args) {
   const id = formatAddress(args.id);
   const query = `
-    SELECT
-      u.*,
-      SUM(l.vote_count) as votesCount,
-      SUM(l.proposal_count) as proposalsCount,
-      MAX(l.last_vote) as lastVote
-    FROM users u
-    LEFT JOIN leaderboard l ON l.user = u.id
-    WHERE id = ?
+    SELECT * FROM (
+      SELECT
+        u.*,
+        u.id as userId,
+        COALESCE(SUM(l.vote_count), 0) as votesCount,
+        COALESCE(SUM(l.proposal_count), 0) as proposalsCount,
+        MAX(l.last_vote) as lastVote
+      FROM users u
+      LEFT JOIN leaderboard l ON l.user = u.id
+      GROUP BY u.id
+      UNION
+      SELECT
+        u.*,
+        l.user as userId,
+        COALESCE(SUM(l.vote_count), 0) as votesCount,
+        COALESCE(SUM(l.proposal_count), 0) as proposalsCount,
+        MAX(l.last_vote) as lastVote
+      FROM users u
+      RIGHT JOIN leaderboard l ON l.user = u.id
+      GROUP BY l.user
+    ) AS t
+    WHERE t.userId = ?
     LIMIT 1`;
```

Comment:
Getting too big, can be confusing, is it possible to get both users and the leaderboard and merge them?

Reply:
This is what the query is doing, merge both tables then query it.

This is the mysql way of doing a `FULL OUTER JOIN`, since it does not support it.